t/dsl/send_file.t

```diff
@@ -36,7 +36,7 @@ use File::Spec;
 
     get '/filehandle' => sub {
         open my $fh, "<:raw", __FILE__;
-        send_file( $fh, content_type => 'text/plain' );
+        send_file( $fh, content_type => 'text/plain', charset => 'utf-8' );
```

Comment:
There's a big difference between `utf-8` and `UTF-8`. This might be traced back to some of our Unicode problems.